features/main-page.feature

```diff
@@ -9,6 +9,4 @@ Feature: Stuff on the main page
   Scenario: Search controls
     Then I should see the keyword search box
     And I should see the location search box
-    And I should see some category selectors
-    And I should see the radius selector
```

Comment:
This reflects Jake's recent changes to radius display, not related to the point of this PR